icons/picoctf.svg

```diff
@@ -0,0 +1,12 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" role="img" viewBox="0 0 24 24">
+  <title>picoCTF</title>
+  <defs>
+    <linearGradient id="a">
+      <stop offset="0" style="stop-color:#000;stop-opacity:.99224806"/>
+    </linearGradient>
+    <linearGradient xlink:href="#a" id="b" x1="31.4" x2="107.2" y1="68.65" y2="68.65" gradientTransform="matrix(.96421 0 0 .97183 2.932 2.606)" gradientUnits="userSpaceOnUse"/>
+  </defs>
+  <path d="M31.3 45.1c0-3.4.5-6.5 1.6-9.2 1-2.8 2.5-5.1 4.5-7q3-2.85 7.2-4.5c2.8-1 6-1.6 9.6-1.6 3.7 0 7 .5 9.899 1.6 2.801 1.1 5.2 2.6 7 4.5l36 36c.7-3.5 1.101-7.1 1.101-10.7C108.2 24.3 84 .1 54.1.1S0 24.3 0 54.2s24.2 54.1 54.1 54.1h.1L31.4 85.5V45.1Z" style="fill:#000;fill-opacity:1;stroke-width:2.34100584;stroke-dasharray:none;paint-order:stroke fill markers" transform="matrix(.2126 0 0 .21458 .499 .36)"/>
+  <path fill="#c41230" d="M48.54 51.295c.77.777 1.735 1.457 2.796 1.846 1.06.486 2.217.68 3.374.68s2.315-.194 3.279-.68c1.061-.486 1.928-1.069 2.7-1.846s1.447-1.653 1.929-2.722a7.7 7.7 0 0 0 .674-3.207c0-1.263-.192-2.43-.674-3.498-.482-1.07-1.062-2.041-1.832-2.916-.77-.874-1.736-1.457-2.796-1.943s-2.218-.778-3.375-.778-2.314.292-3.375.778-1.928 1.166-2.796 1.943a9.7 9.7 0 0 0-1.928 2.819c-.483 1.069-.675 2.138-.675 3.304s.192 2.332.675 3.401c.577 1.07 1.252 2.041 2.023 2.819m22.948-20.603c-.097 0 0 0 0 0 1.736 1.846 3.085 4.081 4.05 6.608a24.4 24.4 0 0 1 1.35 8.066c0 3.207-.58 6.123-1.833 8.65-1.252 2.526-2.796 4.567-4.724 6.317-1.832 1.652-3.955 3.012-6.171 3.79-2.218.874-4.34 1.263-6.364 1.263-2.41 0-4.628-.389-6.557-1.166-1.928-.778-3.567-1.944-4.82-3.402v23.033l-13.21 1.943 21.984 22.158c25.262 0 46.282-18.173 51.103-42.177z" style="fill:url(#b);stroke-width:6.34514;stroke-dasharray:none" transform="matrix(.2126 0 0 .21458 .499 .36)"/>
+  <path fill="#fff" d="M16.507 8.02a3.7 3.7 0 0 0-.893-1.438 4.3 4.3 0 0 0-1.488-.966c-.595-.236-1.297-.343-2.105-.343-.765 0-1.445.107-2.04.343a4.1 4.1 0 0 0-1.531.966 3.94 3.94 0 0 0-.957 1.502c-.212.6-.34 1.245-.34 1.974v8.69l2.913-.429v-5.085c.297.343.637.58 1.063.75a3.9 3.9 0 0 0 1.445.258c.447 0 .914-.086 1.404-.279a4.5 4.5 0 0 0 1.36-.836 4.7 4.7 0 0 0 1.042-1.395c.276-.558.403-1.18.403-1.91a4.9 4.9 0 0 0-.276-1.802m-2.849 2.489c-.106.236-.234.429-.425.6-.17.172-.383.322-.595.408-.234.107-.468.15-.723.15s-.51-.043-.744-.15a2.1 2.1 0 0 1-.617-.407 2.1 2.1 0 0 1-.425-.623 1.8 1.8 0 0 1-.149-.75c0-.258.043-.516.15-.73.105-.236.233-.43.424-.623.17-.171.383-.321.617-.429s.468-.171.744-.171c.255 0 .51.064.744.171s.446.258.616.43.298.386.404.643c.107.236.15.494.15.773a2.2 2.2 0 0 1-.17.708"/>

```

Comment:
As you can see here, there are still 3 paths in your file. You will need to combine them and remove extraneous attributes like `fill` and `style`.

Reply:
Thank you @service-paradis. I am very new to svg creation actually. Can you check whether its fine now. Is there an easy way to change the decimal points. Can you guide me in passing the other tests
Thank you

Reply:
Sure, you can take a look at the section "3. Optimize the Icon" from out [contributing guidelines](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#3-optimize-the-icon) to fix precision errors with SVGO tool.

Reply:
Thank you @service-paradis for your guidance. I have tried and it ahs removed the first set of check errors but still these two exist in` Lint `and `Test` package respectively 

1. extraneous Unexpected newline character(s) detected in SVG markup ( I went through the code but no newlines were found)
2.  AssertionError [ERR_ASSERTION]: Expected values to be strictly equal: - Test package 

Could you please go through these

thank you.